core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -740,6 +740,30 @@ public MetricKey build() {
           .setMetricType(MetricType.GAUGE)
           .setIsClusterAggregated(false)
           .build();
+  public static final MetricKey WORKER_BLOCK_REMOVER_BLOCKS_TO_REMOVED_TAKE_COUNT =
+      new Builder(Name.WORKER_BLOCK_REMOVER_BLOCKS_TO_REMOVED_TAKE_COUNT)
+          .setDescription("Total number of asyncBlockRemover take block count")
+          .setMetricType(MetricType.COUNTER)
+          .setIsClusterAggregated(true)
```

Comment:
Change to `.setIsClusterAggregated(false)`?
Our current way of defining cluster aggregated metrics is:
1) Set MetricKey with  `.setIsClusterAggregated(true)` so that this metrics is reported to the leading master
2) Create a MetricKey like `CLUSTER_WORKER_BLOCK_REMOVER_BLOCKS_TO_REMOVED_TAKE_COUNT` to represent the aggregated cluster metric.
3) In `MetricStore.initCounterKeys` add the cluster metric and worker metrics pair into the map

Reply:
OK, Changed to `.setIsClusterAggregated(false)`.